core/src/main/java/com/distkv/core/concepts/DistkvDicts.java

```diff
@@ -1,32 +1,40 @@
 package com.distkv.core.concepts;
 
-import com.distkv.common.utils.Status;
-import java.util.Map;
+import com.distkv.common.exception.DistkvException;
+import com.distkv.rpc.protobuf.generated.DistkvProtocol.DistkvResponse.Builder;
+import com.google.protobuf.Any;
 
-public interface DistkvDicts {
+public interface DistkvDicts<T> extends DistkvBaseOperation<T> {
```

Comment:
Why `DistkvDicts` should be a generic type?

I guess it's fine to use the original `DistkvDicts`.

Reply:
the base operation are drop, put, get. all data type should has this opeartion.

Reply:
i have modify this.